modules/survivors_benefits/app/models/survivors_benefits/saved_claim.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'survivors_benefits/benefits_intake/submit_claim_job'
+require 'pdf_fill/filler'
+
+module SurvivorsBenefits
+  ##
+  # SurvivorsBenefits 21P-534EZ Active::Record
+  # @see app/model/saved_claim
+  #
+  class SavedClaim < ::SavedClaim
+    # Survivors Benefits Form ID
+    FORM = SurvivorsBenefits::FORM_ID
+
+    # the predefined regional office address
+    #
+    # @return [Array<String>] the address lines of the regional office
+    def regional_office
+      ['Department of Veteran Affairs',
```

Comment:
@joshuakfarrar Should this be Veterans?

Reply:
I'll push this up, but it's code copied from elsewhere so it's the same in pensions, medical_expense_reports.